packages/arb-token-bridge-ui/src/hooks/useNetworksRelationship.ts

```diff
@@ -30,13 +31,16 @@ export function useNetworksRelationship({
       destinationChainId: destinationChain.id
     })
 
-    if (_isDepositMode) {
+    const isWithdrawalMode = !(_isDepositMode || isTeleportMode)
+
+    if (_isDepositMode || isTeleportMode) {
       return {
         childChain: destinationChain,
         childChainProvider: destinationChainProvider,
         parentChain: sourceChain,
         parentChainProvider: sourceChainProvider,
         isDepositMode: _isDepositMode,
+        isWithdrawalMode,
```

Comment:
NIT: I'd just have `isWithdrawalMode: false` here (and same with `isWithdrawalMode: true` below)